Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -5363,11 +5361,7 @@ private List<string> GetItemValueNames(int[] itemValueTypes, List<string> withIt
 
                 whereClauses.Add("guid in (select ItemId from ItemValues where ItemValues.CleanValue=A.CleanName AND " + typeClause + ")");
 
-                var typeWhereText = whereClauses.Count == 0 ?
-                    string.Empty :
-                    " where " + string.Join(" AND ", whereClauses);
-
-                itemCountColumnQuery += typeWhereText;
+                itemCountColumnQuery += " where " + string.Join(" AND ", whereClauses);
```

Comment:
this seems to be broken for `whereClauses.Count == 0`

Reply:
It’s never empty, look at the line above

Reply:
LOL, you're right! How did I miss that...